src/ws.h

```diff
@@ -41,6 +41,7 @@ struct ExportableClient {
     virtual void exportBufferResource(struct wl_resource*) = 0;
     virtual void exportLinuxDmabuf(const struct linux_dmabuf_buffer *dmabuf_buffer) = 0;
     virtual void exportShmBuffer(struct wl_resource*, struct wl_shm_buffer*) = 0;
+    virtual ~ExportableClient() = default;
```

Comment:
Nit: the defaulted virtual constructor fits better at the beginning of this struct declaration.